package/lib/serializer.js

```diff
@@ -58,7 +44,8 @@ try {
    };
 
    MsgpackSerializer.prototype.unserialize = function (payload) {
-      return msgpack.decode(payload, {codec: this.codec});
+      // need to encapsulate ArrayBuffer into Uint8Array for msgpack decoding
+      return msgpack.decode(new Uint8Array(payload), {codec: this.codec});
    };
```

Comment:
Could you pls expand why this is needed? The unit test for msgpack serialization works without this ..